ote/resources/public/language/en.edn

```diff
@@ -1404,7 +1404,8 @@ You can also draw the operating area or point to the map with drawing tools. You
                        :if-not-registered "If you have not recently registered with the NAP service, you can ignore this message."}}
 
  :country-list
- {:AF "Afghanistan"
+ {:AA "Select"
```

Comment:
No need for changes, just a minor thougt: while :AA is completely fine, using e.g. :A would distinct it from valid values even more because also the length is different. It will also be even more quaranteed to be placed first in a sorted list.

Reply:
This is really good idea. Changed